web-server-lib/web-server/http/request-structs.rkt

```diff
@@ -57,21 +58,30 @@
   [bindings-assq-all
    (-> bytes? (listof binding?) (listof binding?))]))
 
-;                                        
-;   ;;                      ;;           
-;   ;;                      ;;           
+(module+ private
+  (provide
+   (struct-out request)
+   (struct-out header)
+   headers-assq
+   headers-assq*
+   binding?
+   (struct-out binding:form)
+   make-binding:file/port))
+
+;
+;   ;;                      ;;
+;   ;;                      ;;
```

Comment:
I've included these whitespace changes because it was pain to work around them, but I can redo the commit if anyone cares. Actually, what I'd really like to do is remove these comments because I think they make reading the code harder and I can't really even read the ascii art (and I assume I'm not alone). Thoughts?

Reply:
Go for it :)

Reply:
I'll do this next time I work on this module.